SQLQuery_3(homework).sql

```diff
@@ -0,0 +1,303 @@
+INSERT INTO Visitors
+VALUES ('Friday', 'Cosmin', 'Petrea',1);
+INSERT INTO Visitors
+VALUES ('Thursday', 'Ioana', 'Maria',1);
+INSERT INTO Visitors
+VALUES ('Saturday', 'Cristea', 'Mariana',1);
+INSERT INTO Visitors
+VALUES ('Sunday', 'Mircea', 'Daniela',1);
+INSERT INTO Visitors
+VALUES ('Sunday', 'Alexa', 'Ioan',1);
+INSERT INTO Visitors
+VALUES ('Friday', 'Florina', 'Petre',1);
+INSERT INTO Visitors
+VALUES ('Sunday', 'Cristi', 'Ion',1);
+
+INSERT INTO Tickets
+VALUES ('one day', 20, '2023-01-20', '2023-01-21', 2,1);
+INSERT INTO Tickets
+VALUES ('one day', 20, '2023-01-20', '2023-01-21', 1,1);
+INSERT INTO Tickets
+VALUES ('one week', 100, '2023-01-20', '2023-01-27', 3,1);
+INSERT INTO Tickets
+VALUES ('one month', 300, '2023-02-01', '2023-03-01', 4,1);
+INSERT INTO Tickets
+VALUES ('one day', 20, '2023-02-12', '2023-02-13', 5,1);
+INSERT INTO Tickets
+VALUES ('one day', 20, '2023-02-05', '2023-02-06', 6,1);
+INSERT INTO Tickets
+VALUES ('one week', 20, '2023-03-12', '2023-03-19', 7,1);
+INSERT INTO Tickets
+VALUES ('one month', 20, '2023-03-01', '2023-04-01', 8,1);
+INSERT INTO Tickets
+VALUES ('one day', 20, '2023-02-11', '2023-02-12', 9,1);
+INSERT INTO Tickets
+VALUES ('one week', 20, '2023-03-02', '2023-03-09', 10,1);
+INSERT INTO Tickets
+VALUES ('one day', 20, '2023-02-18', '2023-02-19', 11,1);
+INSERT INTO Tickets
+VALUES ('one week', 20, '2023-03-22', '2023-03-29', 6,1);
+INSERT INTO Tickets
+VALUES ('one day', 20, '2023-02-26', '2023-02-27', 5,1);
+INSERT INTO Tickets
+VALUES ('one day', 20, '2023-02-26', '2023-02-27', 9,1);
+
+INSERT INTO Animals
+VALUES ('Monkey', 8, 1);
+INSERT INTO Animals
+VALUES ('Tiger', 2, 1);
+INSERT INTO Animals
+VALUES ('Lion', 2, 1);
+INSERT INTO Animals
+VALUES ('Elephant', 1, 1);
+INSERT INTO Animals
+VALUES ('Horse', 3, 1);
+INSERT INTO Animals
+VALUES ('Snake', 6, 1);
+INSERT INTO Animals
+VALUES ('Penguin', 4, 1);
+INSERT INTO Animals
+VALUES ('Giraffe', 2, 1);
+INSERT INTO Animals
+VALUES ('Crocodile', 5, 1);
+
+INSERT INTO Caretakers
+VALUES ('Cristian','Dumitru', 2900, 1);
+INSERT INTO Caretakers
+VALUES ('Florin','Casian', 2500, 1);
+INSERT INTO Caretakers
+VALUES ('Mihai','Popa', 4000, 1);
+INSERT INTO Caretakers
+VALUES ('Catalin','Popescu', 3200, 1);
+INSERT INTO Caretakers
+VALUES ('Narcis','Andrei', 3700, 1);
+INSERT INTO Caretakers
+VALUES ('Robert','Amariei', 3000, 1);
+
+INSERT INTO Caretakers_Animals
+VALUES (1,'Monkey');
+INSERT INTO Caretakers_Animals
+VALUES (1,'Snake');
+INSERT INTO Caretakers_Animals
+VALUES (2,'Horse');
+INSERT INTO Caretakers_Animals
+VALUES (3,'Lion');
+INSERT INTO Caretakers_Animals
+VALUES (3,'Tiger');
+INSERT INTO Caretakers_Animals
+VALUES (4,'Elephant');
+INSERT INTO Caretakers_Animals
+VALUES (5,'Penguin');
+INSERT INTO Caretakers_Animals
+VALUES (5,'Giraffe');
+INSERT INTO Caretakers_Animals
+VALUES (6,'Crocodile');
+
+--Number of total animals group by zoo name
+
+SELECT zoo.name AS 'Zoo Name', SUM(a.NumberOfSpecies)'Total Animals'
```

Comment:
I appreciate that you did all these exercises ahead even they were not required. Great job!